omi/firmware/devkit/prj_xiao_ble_sense_devkitv2-adafruit.conf

```diff
@@ -23,6 +23,14 @@ CONFIG_BT_MAX_PAIRED=1
 CONFIG_BT_DEVICE_APPEARANCE=22
 CONFIG_BT_GATT_DYNAMIC_DB=y
 
+#
+# Preferred connection parameters
```

Comment:
@AnkushMalaker hi. Can you tell, how do you setup and set this params? Which tool or guideline do you use for this?